tests/baselines/reference/typeTagCircularReferenceOnConstructorFunction.errors.txt

```diff
@@ -1,14 +1,11 @@
 tests/cases/conformance/jsdoc/bug27346.js(2,4): error TS8030: The type of a function declaration must match the function's signature.
-tests/cases/conformance/jsdoc/bug27346.js(2,11): error TS2587: JSDoc type 'MyClass' circularly references itself.
 
 
-==== tests/cases/conformance/jsdoc/bug27346.js (2 errors) ====
+==== tests/cases/conformance/jsdoc/bug27346.js (1 errors) ====
     /**
      * @type {MyClass}
        ~~~~~~~~~~~~~~~
 !!! error TS8030: The type of a function declaration must match the function's signature.

```

Comment:
I guess the second error goes away because `getJSClassType` doesn't have `const valueType = getTypeOfSymbol()` any more? Not entirely sure.

Reply:
Actually, it might be worthwhile to check whether we have any other error baselines that include the "JSDoc type circularly references itself".